src/composables/useClient.ts

```diff
@@ -23,7 +23,7 @@ export function useClient() {
       connectorName.value === 'gnosis'
   );
 
-  const usePersonalSign = computed(() => isGnosisSafe.value);
+  const usePersonalSign = computed(() => false);
```

Comment:
This will send a request to the hub right?

Reply:
Not sure what you mean. The change you made here will just use EIP712 instead of personal sign for Gnosis safe.

Reply:
I mean it send sign to hub instead of relayer 
![image](https://user-images.githubusercontent.com/15967809/199701539-1c845b8b-c66f-4811-88e3-c1aa8488c68d.png)

we should send it to the relayer when it is from gnosis right?

Reply:
We should send to relayer if its a Safe yes, maybe it goes to the hub just bcuz the VITE_RELAYER_URL is not set

Reply:
It is `https://relayer.snapshot.org` on fleek , but still it goes to the hub. (goes to hub on vercel preview too, but can't check config there)

Reply:
It is always using HUB url here https://github.com/snapshot-labs/snapshot/blob/samuv/eip712-only/src/helpers/clientEIP712.ts